internal/csi-addons/rbd/reclaimspace.go

```diff
@@ -37,12 +37,14 @@ import (
 // of CSI-addons reclaimspace controller service spec.
 type ReclaimSpaceControllerServer struct {
 	*rs.UnimplementedReclaimSpaceControllerServer
+	// Embed ControllerServer as it implements helper functions
+	*rbdutil.ControllerServer
```

Comment:
It would be cleaner to pass `*util.VolumeLocks` and have that as an attribute of this struct.